### ISSUE TITLE
write privacy considerations

### ISSUE THREAD

**Issue by npdoty** [WICG/digital-credentials#183]
Placeholder section currently points to self-review questionnaire, but not substantive analysis, overview of privacy-related topics throughout the specification or recommendations (or summary of normative recommendations elsewhere).

**Comment by wseltzer** [comment]
Under discussion in https://github.com/WICG/digital-credentials/pull/189

**Comment by timcappalli** [comment]
Removing cgr1-blocker as initial text was added in #189. Leaving open as we still have work to do.

**Comment by wseltzer** [comment]
The editor's draft has section headers with issue labels and text hints:

 [15.Privacy Considerations](https://w3c-fedid.github.io/digital-credentials/#privacy-considerations)
[15.1Unnecessary Requests for Credentials](https://w3c-fedid.github.io/digital-credentials/#unnecessary-requests-for-credentials)
[15.2Over Collection of Data](https://w3c-fedid.github.io/digital-credentials/#over-collection-of-data)
[15.3Individual Consent](https://w3c-fedid.github.io/digital-credentials/#individual-consent)
[15.4Data Retention](https://w3c-fedid.github.io/digital-credentials/#data-retention)
[15.5Compliance with Privacy Regulations](https://w3c-fedid.github.io/digital-credentials/#compliance-with-privacy-regulations)
[15.6Selective and Unlinkable Disclosure](https://w3c-fedid.github.io/digital-credentials/#selective-and-unlinkable-disclosure)
[15.7Phoning Home](https://w3c-fedid.github.io/digital-credentials/#phoning-home)
[15.8Transmission of Personally Identifiable Information](https://w3c-fedid.github.io/digital-credentials/#transmission-of-personally-identifiable-information)

Would it help to make some of those into distinct github issues? 

@npdoty , how much do you think needs to be added pre-publication of FPWD, versus a commitment to address before CR?

**Comment by johannhof** [comment]
I was told some support would be needed here and I'm happy to help write privacy / security considerations. I'll try to get input from @martinthomson and @npdoty, and I'll join the WG call today.

**Comment by johannhof** [comment]
That sounds like a lot of issues, I'm wondering why we need to discuss the latter two separately from the placeholder headings - would you imagine these becoming separate sections as well?

**Comment by johannhof** [comment]
(Otherwise I'd argue we should keep discussion focused on what will end up in this document, and pull in these other (great!) resources as needed)

**Comment by npdoty** [comment]
Yes, I think it's a lot of issues. I would be very surprised if a spec of this kind wasn't mostly text about the potential abuses and mitigations compared to the relatively small amount of API design surface.

We could also refer out to other documents instead and just summarize important conclusions in the privacy considerations here. One alternative was for this WG to adopt a deliverable that established a Threat Model and would include/subsume those other two documents. And once that document was progressed, then we could summarize the important conclusions in a shorter privacy considerations section in the digital-credentials spec. But it sounded like some implementers thought the timeline was too urgent to do that, so I'm willing to in parallel raise separate issues in this spec, if that's what the WG would prefer.

**Comment by johannhof** [comment]
Ok - I agree that it's not unexpected to have a ton of considerations in the spec - I just want to make sure the issues are actionable. So, from my perspective, if you have high-level concern headlines that are not in the spec right now, feel free to file additional issues for those and we can work on getting them in. From a quick glance I'm not sure it needs to be literally every header from those two documents, but then again I'm just ramping up to this :)

**Comment by npdoty** [comment]
Per 25 June call, I think we have many substantive issues on privacy considerations at this point, and substantial text (and a couple open PRs that we believe we can merge in some form shortly). While there's still a lot more privacy work to be done, I don't think we need to maintain this placeholder overview issue to ensure that we write privacy text.

**Comment by johannhof** [comment]
+1 from me to everything said, should we wait on #262 and #283 before closing this?
